src/dataloaderinterface/forms.py

```diff
@@ -209,7 +209,7 @@ class Meta:
 
 class SiteAlertForm(forms.ModelForm):
     notify = forms.BooleanField(required=False, initial=False, label='Notify me if site stops receiving sensor data')
-    hours_threshold = forms.IntegerField(min_value=1, label='Notify after ')
+    hours_threshold = forms.IntegerField(required=False, min_value=1, label='Notify after ')
```

Comment:
@jcaraballo17 I made this field optional so that back end validations would work when the checkbox is left unchecked. Front end validations are still in place to make the input required when the checkbox is checked.